packages/type-definitions/src/index.ts

```diff
@@ -31,6 +31,7 @@ import { types10900 } from './types_10900.js'
 import { calls as didCalls } from './runtime/did.js'
 import { calls as stakingCalls } from './runtime/staking.js'
 import { calls as publicCredentialsCalls } from './runtime/publicCredentials.js'
+import { types10410 } from './types_10410.js'
```

Comment:
🫣 I know it's petty, but can you move that to the other type imports?

Reply:
Did it! [b1f8a41](https://github.com/KILTprotocol/sdk-js/pull/687/commits/b1f8a41bd410da38930de2eb5a2178c0c5065075)